spec/actions_specs/xcodebuild_spec.rb

```diff
@@ -272,7 +337,7 @@
         end").runner.execute(:test)
 
         expect(result).to eq(
-          "set -o pipefail && xcodebuild clean | xcpretty --color --simple"
+          "set -o pipefail && xcodebuild clean | tee '#{File.expand_path("~/Library/Logs/fastlane/xcbuild/#{Time.now.strftime('%F')}/#{Process.pid}/xcodebuild.log")}' | xcpretty --color --simple"
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.